pallets/pips/src/lib.rs

```diff
@@ -1401,9 +1455,27 @@ impl<T: Trait> Module<T> {
     /// Decrement active proposal count if `state` signifies it is active.
     fn decrement_count_if_active(state: ProposalState) {
         if Self::is_active(state) {
-            ActivePipCount::mutate(|count| *count -= 1);
+            ActivePipCount::mutate(|count| *count = count.saturating_sub(1));
```

Comment:
`-=` should not be able to underflow here unless chain state was corrupted.
Did you change this cause of a test or out of caution?

Reply:
Yes, there was a test failure at this point. But it should be fixed by now. Would you like this change reverted?

Reply:
Hmm; perhaps just a clarifying comment would do? The performance impact of a saturating sub shouldn't matter and caution isn't bad.